js/private/translate_pnpm_lock.bzl

```diff
@@ -155,11 +160,16 @@ def _node_bin(rctx):
     return rctx.path(Label("@%s_%s//:bin/node%s" % (rctx.attr.node_repository, host_platform, ".exe" if is_windows_os(rctx) else "")))
 
 def _process_lockfile(rctx):
+    result = rctx.execute([yq_bin(rctx, rctx.attr.yq_repository), "-o=json", ".", rctx.path(rctx.attr.pnpm_lock)])
+    if result.return_code != 0:
+        fail("failed to convert pnpm lockfile to json: %s" % result.stderr)
+    rctx.file("pnpm-lock.json", result.stdout)
```

Comment:
nit: put `"pnpm-lock.json"` in a variable instead of using the static string twice